test/screenshot/spec/mdc-snackbar/fixture.scss

```diff
@@ -20,9 +20,36 @@
 // THE SOFTWARE.
 //
 
-@import "./common";
-@import "../packages/mdc-button/mdc-button";
-@import "../packages/mdc-checkbox/mdc-checkbox";
-@import "../packages/mdc-form-field/mdc-form-field";
-@import "../packages/mdc-snackbar/mdc-snackbar";
-@import "../packages/mdc-textfield/mdc-text-field";
+@import "../../../../packages/mdc-snackbar/mixins";
+@import "../../../../packages/mdc-theme/color-palette";
+@import "../mixins";
```

Comment:
We don't seem to be using this here. Should we remove it?

Reply:
Removed